gallery/src/styles/NxGrid/NxGridExamples.tsx

```diff
@@ -5,10 +5,11 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
+
 import { GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+import { NxCode } from '@sonatype/react-shared-components';
```

Comment:
I see a bunch of places in this PR where you reordered imports.  If you're going to do that, what we really try to do is have imports from external modules at the top.  And that includes, from the gallery's perspective, the RSC library.

Reply:
> I see a bunch of places in this PR where you reordered imports. If you're going to do that, what we really try to do is have imports from external modules at the top. And that includes, from the gallery's perspective, the RSC library.

Initial effort: https://github.com/sonatype/sonatype-react-shared-components/pull/459/commits/a63a568ea2f6e9ca27a8e67773ee8ccfd864826a

Revised effort: https://github.com/sonatype/sonatype-react-shared-components/pull/459/commits/88d3650c12b1cac9aaef616ded454f759b71fabd